rust/definitions/src/navigation.rs

```diff
@@ -13,31 +13,44 @@ pub struct SeedNameWithIdenticon {
     pub identicon: Vec<u8>,
 }
 
+/// A single transaction signing action.
+#[derive(PartialEq, Eq, Debug, Clone)]
+pub struct TransactionSignAction {
+    /// Parsed contents of the transaction.
+    pub content: TransactionCardSet,
+
+    /// If this transaction should be signed with a passworded key.
+    pub has_pwd: bool,
+
+    /// Information about the signing key of this transaction.
+    pub author_info: MAddressCard,
+
+    /// Info about the network this tx happens on.
+    pub network_info: OrderedNetworkSpecs,
+}
+
 /// Enum containing card sets for four different outcomes:
 /// importing derivations (Derivations), signing (Sign),
 /// accepting (Stub) and reading, for example, in case of an error (Read)
 #[derive(PartialEq, Eq, Debug, Clone)]
 pub enum TransactionAction {
     Derivations {
-        content: TransactionCardSet,
-        network_info: OrderedNetworkSpecs,
+        content: Box<TransactionCardSet>,
```

Comment:
Are there any benefits of making it boxed?

Reply:
Yeah that fixes a `clippy` lint about the hugely different sizes of `enum` variants here. They become different since one of the params becomes `Vec` in this changeset as you may see.